docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 mkdocs-material
-git+https://github.com/iterative/jsmin@python3-only#egg=jsmin
+git+https://github.com/tqdm/jsmin@python3-only#egg=jsmin
```

Comment:
I presume someone deleted iterative's fork. In any case 'tis the same as the tqdm fork.

/gripe in future should really have better basic checks before deleting repos.

Reply:
> in future should really have better basic checks before deleting repo
Yeah agree. Hopefully this would be in the actual repo's description - if it's a dependency of some other project, should be noted

Reply:
Sorry for the `jsmin` repository deletion. I haven't found any occurrence in Slack and it was probably accidental, as part of some organization cleanup effort. 🙈